controllers/game.controller.js

```diff
@@ -0,0 +1,137 @@
+const connect = require("../configs/database");
+
+
+//[GET] /games
+module.exports.game = async (req, res) => {
```

Comment:
Change the function name to have more meaningful descriptions (e.g., getListGame)